pysidt/extensions.py

```diff
@@ -82,7 +82,7 @@ def get_extension_edge(
 
     iter = 0
 
-    while grps[iter] != []:
```

Comment:
Are we sure grps[iter] can never be None?

Reply:
`while None` also counts as `False`

Reply:
```
>>> bool(None)
False
```

Reply:
Yes, which mean that if None => False, but if None != [] => True. So there would be different behavior.

Reply:
I've seen that modification create new bugs in other situations.

Reply:
hmmm interesting. Ok I will change it back.